### PR TITLE
[docs] fix typo in fault-tolerance production guide

### PR DESCRIPTION
**Pull request by marwan116**
Fixes a very small misstatement in the serve fault-tolerance docs guide

### CODE REVIEW

File: doc/source/serve/production-guide/fault-tolerance.md [hunk 1]
```diff
@@ -592,7 +592,7 @@ $ python
 
 ### HTTPProxy failure
 
-You can simulate HTTPProxy failures by manually killing deployment replicas. If you're running KubeRay, make sure to `exec` into a Ray pod before running these commands.
+You can simulate HTTPProxy failures by manually killing HTTPProxy actors. If you're running KubeRay, make sure to `exec` into a Ray pod before running these commands.
```
Comment (GeneDer) [inline doc/source/serve/production-guide/fault-tolerance.md:595]: Thanks for fixing, maybe just call it `ProxyActor` to be exact?

File: doc/source/serve/production-guide/fault-tolerance.md [hunk 2]
```diff
@@ -592,7 +592,7 @@ $ python
 
 ### HTTPProxy failure
```
Comment (GeneDer) [inline doc/source/serve/production-guide/fault-tolerance.md:593]: Maybe update this to just Proxy as well

File: doc/source/serve/production-guide/fault-tolerance.md (hunk 1 above)
Comment (marwan116) [reply doc/source/serve/production-guide/fault-tolerance.md]: should now be addressed

File: doc/source/serve/production-guide/fault-tolerance.md (hunk 2 above)
Comment (marwan116) [reply doc/source/serve/production-guide/fault-tolerance.md]: done - I also updated the sentence underneath to state Proxy only

**Review by GeneDer — APPROVED** [review]
:shipit:

**Review by edoakes — APPROVED** [review]
thanks!

**Pull request merged**